langchain/audio_models/base.py

```diff
@@ -0,0 +1,23 @@
+"""Base class for text to audio models."""
+import base64
+import pathlib
+from abc import ABC, abstractmethod
+from io import BytesIO
+
+from pydantic import BaseModel
+
+
+class AudioBase(BaseModel, ABC):
+    @staticmethod
+    def _read_mp3_audio(audio_path: str) -> str:
+        """Read audio file."""
+        if not pathlib.Path(audio_path).exists():
+            raise ValueError(f"Can't find audio file at {audio_path}")
+        assert audio_path.endswith(".mp3"), "Only mp3 files are supported."
```

Comment:
nit: use a check and raise ValueError (as above) rather than asserts for user facing (eg non test code)